data/dtc121.tablet

```diff
@@ -0,0 +1,23 @@
+# Wacom
+# DTC121
+# DTC121
+#
+# sysinfo.DhX8ny40ez.tar.gz
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/327
+
+[Device]
+Name=DTC121
```

Comment:
The other DTCs are Wacom One tablets, is this going to have a similar name? If so it'd be better to put the name of the box into it. If not, let's at least use `Wacom DTC121` please to match the various other PenDisplays

Reply:
The actual name of the product is DTC121 yes, I made the changes requested to match other pen displays. I also have a similar tablet DTH134 added alongside it now with the requested naming style. If you have any other needed changes let me know.